### ISSUE TITLE
[iOS] Support for background image view

### ISSUE THREAD

**Issue by bradzickafoose** [zoontek/react-native-bootsplash#463]
### Why it is needed?

Before upgrading from `react-native-bootsplash` 4.5 to 4.7.3, React Native Bootsplash (officially/unofficially?) supported multiple views within a storyboard on iOS. 

For example, in my app, we have a launch screen storyboard that contains a background image view behind and a logo image view, where the background image view automatically scales to the device's full viewport height/width. However, since updating React Native Bootsplash to the latest release (4.7.3), the native iOS launch screen still shows the background, as intended, but whenever the Bootsplash loading view displays, the background image view is constrained to the width of the logo, and not to the device's viewport dimensions.

I was able to hack together a solution and patch the package, but was curious if there were plans to support background image views in the future?

### Possible implementation

This is a naive solution but maybe implementing a conditional in the `initWithStoryboard` with the following code could work?

```
  UIImageView *backgroundImageView = [[UIImageView alloc] initWithFrame:_loadingView.bounds];
  backgroundImageView.autoresizingMask = UIViewAutoresizingFlexibleWidth | UIViewAutoresizingFlexibleHeight;
  backgroundImageView.contentMode = UIViewContentModeScaleAspectFill;
  backgroundImageView.image = [UIImage imageNamed:@"splash-background-image"];
  [_loadingView addSubview:backgroundImageView];
```

**Comment by zoontek** [comment]
@bradzickafoose It has never been officially supported (no mention in the README), but there's nothing specific to this package here: you are free to open the storyboard in Xcode and edit it from there.

**Comment by zoontek** [comment]
@bradzickafoose It might be related to the orientation fix in `4.7.1`: https://github.com/zoontek/react-native-bootsplash/releases/tag/4.7.1

But I don't get why the storyboard changes are not kept (as bootplash only recreate an instance of the storyboard). Could you create a small reproduction repository?

**Comment by zoontek** [comment]
@bradzickafoose I think there's no need for this still, would love a quick repro to help you debug this (maybe see https://stackoverflow.com/questions/32823380/how-to-make-an-item-full-width-in-auto-layout-constraint)
